examples/browser/npm-shrinkwrap.json

```diff
@@ -7183,9 +6559,9 @@
       "dev": true
     },
     "xterm": {
-      "version": "2.7.0",
+      "version": "2.8.1",
       "from": "xterm@>=2.6.0 <3.0.0",
-      "resolved": "https://registry.npmjs.org/xterm/-/xterm-2.7.0.tgz"
+      "resolved": "https://registry.npmjs.org/xterm/-/xterm-2.8.1.tgz"
     },
     "y18n": {
       "version": "3.2.1",
```

Comment:
Electron is not the only change in the shrinkwrap here... it is not good.
Strange thing it looks like the changes from the previous commit are in this one also ?